LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Niek Palm
+Copyright (c) 2018 The terraform-docs Authors.
```

Comment:
Can you revert this change, was it changed by terraform-docs?

Reply:
Oh yes, of course. I didn't do that manually.